webplugin/js/app/km-utils.js

```diff
@@ -286,7 +283,7 @@ KommunicateUtils = {
             path +
             (secure ? ';secure' : '') +
             (domain ? ';domain=' + domain : '') +
-            (isChrome ? ';samesite=lax' : '');
+            (';SameSite=Strict');
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Document/cookie
![Screenshot 2022-07-07 at 12 42 49 PM](https://user-images.githubusercontent.com/19753380/177714097-c813cb5e-48be-4dfd-9396-48a45a7c6ce8.png)

should it be SameSite or samesite?

Reply:
done